spring-integration-jdbc/src/main/sql/schema.sql.vpp

```diff
@@ -56,6 +56,7 @@ CREATE INDEX INT_CHANNEL_MSG_PRIORITY_IDX ON INT_CHANNEL_MESSAGE (MESSAGE_PRIORI
 
 CREATE TABLE INT_METADATA_STORE  (
 	METADATA_KEY ${VARCHAR}(255),
-	METADATA_VALUE ${VARCHAR}(255),
+	METADATA_VALUE ${VARCHAR}(4000),
+	REGION ${VARCHAR}(100) NOT NULL,
 	constraint METADATA_STORE primary key (METADATA_KEY)
```

Comment:
After introducing `REGION` it must become as a part of PK. 
See similar state on the `INT_LOCK` table.